operator/config/manifests/bases/ibm-spectrum-scale-csi-operator.clusterserviceversion.yaml

```diff
@@ -21,277 +29,260 @@ metadata:
+        path: attacherNodeSelector[0].value
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:label
-      - description: The cluster id of IBM Spectrum Scale cluster specified (mandatory).
-        displayName: ID
-        path: clusters.id
+      - description: clusters is a collection of GPFS CSICluster properties for the
+          CSI driver to mount.
+        displayName: Clusters
+        path: clusters
+      - description: cacert is the name of the configMap storing GUI certificates.
+          Mandatory if secureSslMode is true.
+        displayName: CA Certificate Resource Name
+        path: clusters[0].cacert
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes:ConfigMap
+      - description: id is the CSICluster id of the GPFS CSICluster specified.
+        displayName: Cluster ID
+        path: clusters[0].id
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: primary is the primary file system for the GPFS CSICluster
+        displayName: Primary
+        path: clusters[0].primary
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:label
-      - description: A collection of targets for REST calls.
+      - description: restApi is a collection of targets for REST calls
         displayName: REST API
-        path: clusters.restApi
+        path: clusters[0].restApi
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:label
-      - description: The hostname of IBM Spectrum Scale REST server.
+      - description: The hostname of the REST server.
         displayName: GUI Host
-        path: clusters.restApi.guiHost
+        path: clusters[0].restApi[0].guiHost
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:label
-      - description: The port number running the IBM Spectrum Scale REST server.
+      - description: The port number running the REST server.
         displayName: GUI Port
-        path: clusters.restApi.guiPort
+        path: clusters[0].restApi[0].guiPort
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:number
-      - description: A cacert resource name for IBM Spectrum Scale REST server SSL certificates
-        displayName: cacert Resource Name
-        path: clusters.cacert
+      - description: secret is the name of the basic-auth secret containing credentials
+          to connect to IBM Spectrum Scale REST API server.
+        displayName: Secrets
+        path: clusters[0].secrets
         x-descriptors:
-        - urn:alm:descriptor:com.tectonic.ui:label
-      - description: Require a secure SSL connection to connect to IBM Spectrum Scale  REST server
+        - urn:alm:descriptor:io.kubernetes:Secret
+      - description: secureSslMode specifies if a secure SSL connection to connect
+          to GPFS is required.
         displayName: Secure SSL Mode
-        path: clusters.secureSslMode
+        path: clusters[0].secureSslMode
```

Comment:
Will this work for remote clusters?

Reply:
yes, it should have not impact on remotecluster, though the controller code is not fully capable to handle multiple values under spec.cluster as I can see error in operator logs.

Reply:
If only remotecluster details are passed, it should be just fine. Tested it.

Reply:
Regarding path field, this used by OCP UI to display details from CR in UI, this is implemented correctly now. in previous versions this was not implemented correctly.